src/xrpld/app/misc/NetworkOPs.cpp

```diff
@@ -1472,16 +1583,54 @@ NetworkOPsImp::apply(std::unique_lock<std::mutex>& batchLock)
                 e.transaction->setQueued();
                 e.transaction->setKept();
             }
-            else if (isTerRetry(e.result))
+            else if (
```

Comment:
again, this I think could cause a difference in transactions sets, making me wonder how safe is this change without an amendment.

Reply:
> again, this I think could cause a difference in transactions sets, making me wonder how safe is this change without an amendment.

Again, this is in the context of the open ledger, which won't affect consensus or the validated ledger.